cmake/ConfigureInstall.cmake

```diff
@@ -14,14 +14,7 @@ list(APPEND CMAKE_PREFIX_PATH "${GENNY_INSTALL_DIR}")
 
 include(GNUInstallDirs)
 
-# This line allows us to run genny executables from a distribution folder where libraries live in
-# bin/../lib64. This is a nice thing for driver-like applications, because you can pass around
-# archives trivially.
-set(CMAKE_INSTALL_RPATH                 "\$ORIGIN/../${CMAKE_INSTALL_LIBDIR}"   CACHE PATH "")
```

Comment:
This will break DSI. At the least, keep the ORIGIN if not also doing `@executable_path` (I've only done that via the macos post link tools. I think it should work.)

Some helpful links:
https://wincent.com/wiki/%40executable_path%2C_%40load_path_and_%40rpath
https://stackoverflow.com/questions/40146437/how-to-set-multiple-rpath-directories-using-cmake-on-macos

Reply:
1. so it compiles on arch and alinux2 without this line, but I haven't tested running it from a genny patch-build yet.
2. BUT with this line in there the `dist/bin/genny` version doesn't work on osx. The `build/src/driver/genny` version works but not when it's moved to `dist`.

```sh
# dist version with the CMAKE_INSTALL_RPATH line
$> ../dist/bin/genny
dyld: Library not loaded: @rpath/libcast_core.dylib
  Referenced from: /Users/rtimmons/CLionProjects/squeeze/build/../dist/bin/genny
  Reason: image not found
zsh: abort      ../dist/bin/genny
```

Reply:
Yeah, so it will work in install before it's redistributed but not in a different directory. This is because `$ORIGIN` is the only way linux knows about relative library paths. `@executable_path`/`@rpath` is the macOS version of that. Not that your `rpath` there isn't relative ala `@rpath/../lib`. I suspect that it's due to `CMAKE_MACOSX_RPATH`.